external-builds/pytorch/patches/pytorch/main/pytorch/hipified/0002-Support-FLASH_ATTENTION-MEM_EFF_ATTENTION-via.-aotri.patch

```diff
@@ -357,7 +357,7 @@ index 54564e42c90..fa9beea8026 100644
 +        file(REMOVE_RECURSE "${__AOTRITON_TARBALL_EXTRACT_DIR}")
 +        file(MAKE_DIRECTORY "${__AOTRITON_TARBALL_EXTRACT_DIR}")
 +        execute_process(
-+          COMMAND "${CMAKE_COMMAND}" -E tar xf "${__AOTRITON_TARBALL_PATH}"
++          COMMAND tar --options hdrcharset=UTF-8 -xf "${__AOTRITON_TARBALL_PATH}"
```

Comment:
I tested these commands outside of CMake and the `--options hdrcharset=UTF-8` is required for successful extraction:

<img width="768" height="231" alt="Image" src="https://github.com/user-attachments/assets/925fdbfd-616f-48aa-a4c8-3b87717b45da" />

top files are with the flag, bottom are without.

My local PyTorch builds are broken for unrelated reasons so I can't completely test this right now, but I think that's enough of a test to say this looks good.